configs/NanoV9Data2018_cfg.py

```diff
@@ -0,0 +1,3059 @@
+# Auto generated configuration file
+# using: 
+# Revision: 1.19 
+# Source: /local/reps/CMSSW/CMSSW/Configuration/Applications/python/ConfigBuilder.py,v 
+# with command line options: RECO --conditions 106X_dataRun2_v36 --customise Configuration/DataProcessing/Utils.addMonitoring --datatier NANOAOD --era Run2_2018,run2_nanoAOD_106Xv2 --eventcontent NANOAOD --filein dbs:/SingleMuon/Run2018A-UL2018_MiniAODv2_GT36-v2/MINIAOD --fileout file:NanoV9Data2018.root --nThreads 1 --no_exec --number 1000 --python_filename configs/NanoV9Data2018_cfg.py --scenario pp --step NANO --data
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run2_2018_cff import Run2_2018
+from Configuration.Eras.Modifier_run2_nanoAOD_106Xv2_cff import run2_nanoAOD_106Xv2
+
+process = cms.Process('NANO',Run2_2018,run2_nanoAOD_106Xv2)
+
+# import of standard configurations
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('SimGeneral.HepPDTESSource.pythiapdt_cfi')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+process.load('Configuration.EventContent.EventContent_cff')
+process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
+process.load('Configuration.Geometry.GeometrySimDB_cff')
+process.load('Configuration.StandardSequences.MagneticField_AutoFromDBCurrent_cff')
+process.load('PhysicsTools.NanoAOD.nano_cff')
+process.load('Configuration.StandardSequences.EndOfProcess_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(1000)
+)
+
+# Input source
+process.source = cms.Source("PoolSource",
```

Comment:
This apparently contains only data from era A. Does 2018 have only that one ? Otherwise, what's the purpose of having the whole list of files here ?

Reply:
The list gets overwritten on submission, though frankly one single file for testing would be enough to have.

Reply:
Reduced to three files.